assets/css/style.css

```diff
@@ -50,16 +44,23 @@ h1, h2, h3, h4, h5, h6 {
 }
 
 .nav{
-    font-size: 20px;
+    font-size: 100%;

```

Comment:
nah) if u are changing ur font-size, then it's better to use rem, 1 rem = font-size of the page, it's usually 16px.